webapp-django/crashstats/settings/bundles.py

```diff
@@ -228,8 +228,6 @@
             'crashstats/js/jquery/plugins/jquery.cookies.2.2.0.js',
             'crashstats/js/socorro/nav.js',
             'crashstats/js/socorro/analytics.js',
-            'browserid/api.js',
-            'browserid/browserid.js',
```

Comment:
Haha, I can't believe we didn't catch this... :) (I was working on this in a train with no internet connection. )